src/System.Runtime/ref/System.Runtime.csproj

```diff
@@ -6,15 +6,22 @@
     <AssemblyVersion>4.2.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
     <IsCoreAssembly>true</IsCoreAssembly>
-    <PackageTargetFramework>netstandard1.7</PackageTargetFramework>
     <NuGetTargetMoniker>.NETStandard,Version=v1.5</NuGetTargetMoniker>
   </PropertyGroup>
   <ItemGroup>
+    <PackageDestination Include="ref/netstandard1.7">
+      <TargetFramework>netstandard1.7</TargetFramework>
+    </PackageDestination>
+    <PackageDestination Include="ref/uap10.1">
+      <TargetFramework>uap10.1</TargetFramework>
+    </PackageDestination>
+  </ItemGroup>
```

Comment:
Same comment than in System.Private.Uri, lets try to use `<PackageTargetFramework>` property instead.